templates/zonefile.j2

```diff
@@ -23,7 +23,7 @@ api-int		IN	A	{{ helper.ipaddr }}
 ;
 {%if setup_registry %}
 ; Create entry for the local registry
-{{ registry_host }}	IN	A	{{ helper.ipaddr }}
+registry	IN	A	{{ helper.ipaddr }}
```

Comment:
I think setting this to a `CNAME` to the helper. Maybe to `{{ helper.name | lower }}.{{ dns.clusterid }}.{{ dns.domain | lower }}.` should be enough.

Reply:
Change to use helper.name is ok, but we may need to change several places where registry name is used. What do you think?

Reply:
@cs-zhang Looks like this was already addressed in #94 ...so this part isn't needed yet.